src/wp-includes/block-template-utils.php

```diff
@@ -725,74 +725,115 @@ function _wp_build_title_and_description_for_taxonomy_block_template( $taxonomy,
 }
 
 /**
- * Builds a unified template object based a post Object.
+ * Build a block template object from a post object.
```

Comment:
Nitpick: 3rd person form is used in the code comments:
```suggestion
 * Builds a block template object from a post object.
```